### PR TITLE
fix(server): log API route errors

### PR DESCRIPTION
**Pull request by Industrial**

### CODE REVIEW

File: src/resolver.ts [hunk 1]
```diff
@@ -33,3 +33,7 @@ export const hashFile = (url: string) => {
   );
   return hashHex;
 };
+
+export function ensureNoTrailingSlash(url: string) {
```
Comment (dburles) [inline src/resolver.ts:37]: Maybe name this `stripTrailingSlash` as I feel `ensure` implies that the function throws.

File: src/server.ts [hunk 2]
```diff
@@ -113,13 +113,19 @@ const server = () => {
...
-        return await (await importAPIRoute(pathname))(request);
-      } catch (_error) {
-        return new Response(`Not found`, { status: 404 });
+        const pathname = ensureNoTrailingSlash(url.pathname);
+        const handler = await importAPIRoute(pathname);
+        const response = await handler(request);
+        return response;
+      } catch (error) {
+        console.error(error);
+        return new Response(`Internal Server Error`, {
```
Comment (dburles) [inline src/server.ts:123]: Can just be string instead of template literal.

**Review by dburles — CHANGES_REQUESTED** [review]

File: src/resolver.ts (hunk 1 above)
Comment (Industrial) [reply src/resolver.ts]: Done

File: src/server.ts (hunk 2 above)
Comment (Industrial) [reply src/server.ts]: Done.

**Pull request merged**